flink-runtime/src/test/java/org/apache/flink/runtime/jobmaster/slotpool/TestingSlotPoolImpl.java

```diff
@@ -39,6 +43,10 @@
 
 	private ResourceProfile lastRequestedSlotResourceProfile;
 
+	private volatile Consumer<SlotRequestId> releaseSlotConsumer;
```

Comment:
Could we have a separate commit to merge `SlotPoolInteractionsTest#TestingSlotPool` with `TestingSlotPoolImpl`?
It can be easier to reasoning which methods are ported instead of being newly added.

Reply:
I get it. I'll do it when I rearrange the commits.